docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -1,116 +1,252 @@
+
+We recommend using [Conda](https://docs.conda.io/en/latest/miniconda.html) to create a python 3.9 enviroment:
 
-For optimal performance with LLM models using BigDL-LLM optimizations on Intel GPUs, here are some best practices for setting up environment:
+```eval_rst
+.. important::
 
-First we recommend using [Conda](https://docs.conda.io/en/latest/miniconda.html) to create a python 3.9 enviroment:
+   ``bigdl-llm`` is tested with Python 3.9, which is recommended for best practices.
+```
 
 ```bash
 conda create -n llm python=3.9
 conda activate llm
 
-pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu # install bigdl-llm for GPU
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
 ```
 
-If you encounter network issues when installing ipex, you can refer to  [Installing bigdl-llm[xpu] dependencies from Wheels](#installing-bigdl-llm[xpu]-dependencies-from-wheels) as an alternative method.
-
-Then for running a LLM model with BigDL-LLM optimizations, several environment variables are recommended:
 
-```bash
-# configures OneAPI environment variables
-source /opt/intel/oneapi/setvars.sh
+```eval_rst
+.. note::
 
-export USE_XETLA=OFF
-export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+   BigDL-LLM with PyTorch 2.0 only supports Ubuntu 20.04 operating system or later.
 ```
 
-## Installing bigdl-llm[xpu] dependencies from Wheels
+## Known issues
+### 1. For Linux users, Ubuntu 22.04 and Linux kernel 5.19.0 is prefered
 
-You can also install BigDL-LLM dependencies for Intel XPU from source achieves. First you need to install the target torch/torchvision/ipex versions from downloaded whls [here](http://ec2-52-27-27-201.us-west-2.compute.amazonaws.com/ipex-release.php?device=xpu&repo=us&release=stable) before installing bigdl-llm. 
+Ubuntu 22.04 and Linux kernel 5.19.0-41-generic is mostly used in our test environment. But default linux kernel of ubuntu 22.04.3 is 6.2.0-35-generic, so we recommonded you to downgrade kernel to 5.19.0-41-generic to archive the best performance.
 
-```bash
-# get the wheels on Linux system
-wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torch-2.0.1a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
-wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.15.2a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
-wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.0.110%2Bxpu-cp39-cp39-linux_x86_64.whl
+### 2. Best known configurations
 
-```
+For running a LLM model with BigDL-LLM optimizations, several environment variables are recommended:
 
 ```eval_rst
-.. note::
+.. tabs::
+   .. tab:: Linux
+
+      .. code-block:: bash
+
+         # configures OneAPI environment variables
+         source /opt/intel/oneapi/setvars.sh
+
+         export USE_XETLA=OFF
+         export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
```

Comment:
For Intel Arc™ A-Series Graphics and Intel Data Center GPU Flex Series, we recommend:
```bash
export USE_XETLA=OFF
export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
```

For Intel Data Center GPU Max Series, we recommend:
```bash
export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libtcmalloc.so
export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
export ENABLE_SDP_FUSION=1
```
`libtcmalloc.so` can installed by `conda install -c conda-forge -y gperftools=2.10`